Makefile

```diff
@@ -270,6 +270,14 @@ deploy_nodes_with_install:
 	bash scripts/utils.sh local_setup_before_deployment $(PLATFORM) $(NAMESPACE) $(OC_FLAG)
 	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) ADDITIONAL_PARAMS="'-in ${ADDITIONAL_PARAMS}'" $(SKIPPER_PARAMS) DAY1_PARAMS=--day1-cluster
 
+deploy_single_node_ipv6_with_install:
+	make deploy_nodes_with_install NUM_MASTERS=1 IPv6=yes
```

Comment:
`IPv4=no`, otherwise many things may work not because they work fine with IPv6, but because IPv4 is there. Also, how does it work without a proxy? Do we support IPv6 without a proxy already?

Reply:
About IPv4, it's a nice catch. I'll change this target. Although for the larger milestone I'm not sure if it's exactly what the user will do (whether it's IPv6 only or both)
About the proxy, let me get back to you

Reply:
I guess you should be adding `PROXY=true` and it will be configured automatically. It's a recent change, see the `run_full_flow_with_ipv6` target.

Reply:
@ronniel1 our flow talks about IPv4+IPv6 or on a pure IPv6 environment?